authorization.adoc

```diff
@@ -75,23 +75,36 @@ The response will contain both the access and refresh tokens:
 }
 ```
 
-### Password Grant
-For private integrations, you should implement the password grant as described below. This will allow you to retrieve an access token for your iZettle account.
+### Assertion Grant
+For private integrations, you should implement the assertion grant as described below. This will allow you to retrieve an access token using an API key provided to you by the iZettle account owner.
 
-*NOTE:* In the future, this process will probably change, allowing you to create an personal access token directly in your merchant dashboard.
+#### Create API key
+The iZettle account owner can create an API key from within the iZettle backoffice. This functionality might not yet be available to all users but is being rolled out in the near future. The API key is provided in the form of a JWT string and serves as the assertion that the integrator will use when calling the assertion grant. It is of highest importance that this key is being kept secure!
```

Comment:
> This functionality might not yet be available to all users but is being rolled out in the near future. 

It's such a small thing, everything's already pretty much been built. I think i'll just go ahead and do it